commands2/__init__.py

```diff
@@ -1,96 +1,87 @@
-    RamseteCommand,
-    RepeatCommand,
-    RunCommand,
-    ScheduleCommand,
-    SelectCommand,
-    SequentialCommandGroup,
-    StartEndCommand,
-    Subsystem,
-    SubsystemBase,
-    Swerve2ControllerCommand,
-    Swerve3ControllerCommand,
-    Swerve4ControllerCommand,
-    Swerve6ControllerCommand,
-    TimedCommandRobot,
-    TrapezoidProfileCommand,
-    TrapezoidProfileCommandRadians,
-    TrapezoidProfileSubsystem,
-    TrapezoidProfileSubsystemRadians,
-    Trigger,
-    WaitCommand,
-    WaitUntilCommand,
-    # button,
-    # cmd,
-    requirementsDisjoint,
+from . import button
+from .command import Command, InterruptionBehavior
+from .commandfactories import (
+    deadline,
+    either,
+    none,
+    parallel,
+    print_,
+    race,
+    repeatingSequence,
+    run,
+    runEnd,
+    runOnce,
+    select,
+    sequence,
+    startEnd,
+    waitSeconds,
+    waitUntil,
 )
+from .commandgroup import CommandGroup, IllegalCommandUse
+from .commandscheduler import CommandScheduler
+from .conditionalcommand import ConditionalCommand
+from .functionalcommand import FunctionalCommand
+from .instantcommand import InstantCommand
+from .notifiercommand import NotifierCommand
+from .parallelcommandgroup import ParallelCommandGroup
+from .paralleldeadlinegroup import ParallelDeadlineGroup
+from .parallelracegroup import ParallelRaceGroup
+from .perpetualcommand import PerpetualCommand
+from .printcommand import PrintCommand
+from .proxycommand import ProxyCommand
+from .proxyschedulecommand import ProxyScheduleCommand
+from .repeatcommand import RepeatCommand
+from .runcommand import RunCommand
+from .schedulecommand import ScheduleCommand
+from .selectcommand import SelectCommand
+from .sequentialcommandgroup import SequentialCommandGroup
+from .startendcommand import StartEndCommand
+from .subsystem import Subsystem
+from .waitcommand import WaitCommand
+from .waituntilcommand import WaitUntilCommand
+from .wrappercommand import WrapperCommand
 
 __all__ = [
+    "button",
     "Command",
-    "CommandBase",
-    "CommandGroupBase",
+    "CommandGroup",
     "CommandScheduler",
     "ConditionalCommand",
-    "SelectCommand",
     "FunctionalCommand",
+    "IllegalCommandUse",
     "InstantCommand",
-    "MecanumControllerCommand",
+    "InterruptionBehavior",
     "NotifierCommand",
-    "PIDCommand",
-    "PIDSubsystem",
     "ParallelCommandGroup",
     "ParallelDeadlineGroup",
     "ParallelRaceGroup",
     "PerpetualCommand",
     "PrintCommand",
-    "ProfiledPIDCommand",
-    "ProfiledPIDSubsystem",
+    "ProxyCommand",
     "ProxyScheduleCommand",
-    "RamseteCommand",
     "RepeatCommand",
     "RunCommand",
     "ScheduleCommand",
+    "SelectCommand",
     "SequentialCommandGroup",
     "StartEndCommand",
     "Subsystem",
-    "SubsystemBase",
-    "Swerve2ControllerCommand",
-    "Swerve3ControllerCommand",
-    "Swerve4ControllerCommand",
-    "Swerve6ControllerCommand",
-    "TimedCommandRobot",
```

Comment:
What happened to TimedCommandRobot?

Reply:
Whoops forgot. Added it in. The java/cpp version isn't merged yet. https://github.com/wpilibsuite/allwpilib/pull/4871